src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/HotReload/ProjectHotReloadSession.cs

```diff
@@ -14,6 +16,10 @@ internal class ProjectHotReloadSession : IManagedHotReloadAgent, IManagedHotRelo
     private readonly Lazy<IHotReloadDiagnosticOutputService> _hotReloadOutputService;
     private readonly Lazy<IManagedDeltaApplierCreator> _deltaApplierCreator;
     private readonly IProjectHotReloadSessionCallback _callback;
+    private readonly IDebugLaunchProvider? _launchProvider;
```

Comment:
We don't use the interface in this class, only the concrete `LaunchProfilesDebugLaunchProvider` type. Can this field be made to use that type directly? Or should a new interface (like `IDebugLaunchProvider2`) be created so that we can unit test this.

Same question for `IProjectHotReloadSessionManager`/`ProjectHotReloadSessionManager`.

Reply:
I think the latter one, maybe creating an `IInternalDebugLaunchProvider`, for unit testing purpose

Same answer for `IProjectHotReloadSessionManager`